content/4_Gold/LIS.mdx

```diff
@@ -587,6 +587,43 @@ class PCB {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+import sys
+
+input = sys.stdin.read
+
+lis = 0
+a = []
+s = set()
+
+
+def main():
```

Comment:
is the main method necessary

Reply:
no, just included it for readability.
i can delete it though.